README.md

```diff
@@ -1,24 +1,49 @@
 # WHR 930
+
 Manage a Storkair WHR 930 system using MQTT.
 
 See also my [blog posting](https://blog.mosibi.nl/domotica/2017/12/31/control-a-storkair-zehnder-whr-930-ventilation-unit-using-mqtt.html) I wrote about the WHR 930
 
+## Installation
+
+This (python) code uses some libraries that need to be installed. Assuming you are using a Raspberry Pi or another Debian based Linux distribution, the following command's will clone the repository and install the required dependencies:
 
-## Installation and configuration
-This (python) code uses some libraries that need to be installed. Assuming you are using a Raspberry Pi or another Debian based Linux distribution, the following command's will install the dependencies
+For installation, you can choose between installing the bridge as a system service or use Docker.
 
-```lang=shell
-$ sudo apt-get install python3-serial python3-pip python3-yaml
-$ sudo pip3 install paho-mqtt
-````
+### Installing as system service
 
-After installing the dependencies, clone this repository and modify the MQTT settings in src/whr930.py and run `sudo make install`.
+Follow these steps to clone the repository, install required dependencies, configure the service and install it:
 
-```lang=shell
-$ git clone https://github.com/Mosibi/whr_930.git
-$ vi src/whr930.py
-$ sudo make install
-$ sudo systemctl start whr930.service
+```bash
+git clone https://github.com/Mosibi/whr_930.git
+sudo apt-get install python3-serial python3-pip python3-yaml
+cd whr_930
+vi src/config.yaml
+sudo pip3 install -r src/requirements.txt
+sudo make install
+sudo systemctl start whr930.service
+```
+
+### Using Docker
+
+Follow these steps to clone the repository, configure the service and run it through Docker.
+
+```bash
+git clone https://github.com/Mosibi/whr_930.git
+cd whr_930
+vi src/config.yaml
+docker build -t wtw-bridge .
+docker start wtw-bridge
```

Comment:
I assume that the container needs privileged rights to be able to open the serial device. If needed, state that in the README.

Reply:
Tested it, using the `--device` flag this does not seem to be the case. However, forgot including that flag, fixed it now.